src/globus_cli/parsing/commands.py

```diff
@@ -74,7 +75,28 @@ def parse_args(self, ctx: click.Context, args: list[str]) -> list[str]:
         # args will be consumed, so check it before super()
         had_args = bool(args)
         try:
+            # if we have any opts to be combined in order, do that now
+            if self.opts_to_combine:
+                combined_opts: dict[str, list[tuple[str, str]]] = {
+                    combined_name: [] for combined_name in self.opts_to_combine.values()
+                }
+                parser = self.make_parser(ctx)
+                values, _, order = parser.parse_args(args=list(args))
+                # values is a dict of value lists keyed by their option name
+                # in order for that value and opts is a list of option names
+                # in the order they were given at the command line
+                # we want a list of (name, value) tuples for multiple options
+                # in the order they were given at the command line
```

Comment:
> `...and opts is a list of option names`

This sentence suggests that a variable named "opts" may have been renamed to "order", which is now seen in the code. I think either the variable or the comment may need to be updated.